app/models/flood_risk_engine/enrollment.rb

```diff
@@ -20,6 +19,8 @@ class Enrollment < ActiveRecord::Base
     # The Correspondence Contact Details related to this Application a.k.a Main Contact
     belongs_to :correspondence_contact, class_name: "Contact"
 
+    has_one :location, as: :locatable, dependent: :restrict_with_exception
```

Comment:
what is this ? the name doesn't really tell us what business need this full fills ... if this is the location where exemption activity takes place then I think `site_address` more meaningful or activity_location ...  I suspect that each Exemption maybe linked to a site so may not even be needed ... future prrofs against multiple exemptions/sites to start of use something like  EnrollmentExemption

```
class EnrollmentExemption < ActiveRecord::Base

    belongs_to :enrollment,
    belongs_to :exemption
    belongs_to :site_address,
 
```